repository_service_tuf_api/metadata.py

```diff
@@ -120,18 +125,31 @@ def get_metadata_sign() -> MetadataSignGetResponse:
         filter(lambda var: "SIGNING" in var, dir(settings_repository))
     )
 
-    metadata_response: Dict[str, TUFMetadata] = {}
-    for role in pending_signing:
-        signing_md = settings_repository.get(role)
-        if signing_md is not None:
-            metadata_response[
-                role.split("_")[0].lower()
-            ] = signing_md.to_dict()
-
-    return MetadataSignGetResponse(
-        data={"metadata": metadata_response},
-        message="Metadata role(s) pending signing",
-    )
+    md_response = {}
+    for role_setting in pending_signing:
+        signing_role_obj = settings_repository.get(role_setting)
+        if signing_role_obj is not None:
+            signing_role_dict = signing_role_obj.to_dict()
+            role = role_setting.split("_")[0].lower()
+            md_response[role] = signing_role_dict
+
+            trusted_obj = settings_repository.get(f"TRUSTED_{role.upper()}")
+            md_response[f"trusted_{role}"] = {}
+            if trusted_obj is not None:
+                trusted_dict = trusted_obj.to_dict()
+                # If versions match, signing_role_obj is initial bootsrap root.
+                signing_ver = signing_role_dict["signed"]["version"]
+                if trusted_dict["signed"]["version"] != signing_ver:
+                    md_response[f"trusted_{role}"] = trusted_dict
```

Comment:
I would leave it to the client API user.
They can apply their rules and handle them with more freedom.